client/my-sites/email/email-management/home/email-list-active.jsx

```diff
@@ -18,9 +18,15 @@ import {
 	resolveEmailPlanStatus,
 } from 'calypso/my-sites/email/email-management/home/utils';
 import MaterialIcon from 'calypso/components/material-icon';
+import { useEmailsQuery } from 'calypso/data/emails/use-emails-query';
 
-const EmailListActiveWarning = ( { domain } ) => {
-	const { icon, statusClass, text } = resolveEmailPlanStatus( domain );
+const EmailListActiveWarning = ( { domain, siteId } ) => {
+	const { data, error, isLoading } = useEmailsQuery( siteId, domain.name, { retry: false } );
+	let emailAccounts = null;
+	if ( ! isLoading && ! error ) {
+		emailAccounts = data?.accounts[ 0 ] ?? [];
+	}
+	const { icon, statusClass, text } = resolveEmailPlanStatus( domain, emailAccounts, isLoading );
```

Comment:
Could we add blank lines? Right now it's just a block of code which is quite dense and harder to read. Thanks!